src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -54,10 +54,11 @@ const abs: CSSProperties = {
   right: '0',
 };
 
-const DEFAULT_INPUT_VALUE = `GET _search
+const DEFAULT_INPUT_VALUE = `# Click the Variables button, above, to create your own variables.
```

Comment:
I just realized we can also use your awesome new inline comments to make this even clearer!

```
# Click the Variables button, above, to create your own variables.
GET ${exampleVariable1} // _search
{
  "query": {
    "${exampleVariable2}": {} // _match_all
  }
}
```